popmon/visualization/utils.py

```diff
@@ -126,40 +110,31 @@ def plot_bars_b64(data, labels=None, bounds=None, ylim=False, skip_empty=True):
             y_max += spread
             y_min -= spread
 
-            yellow = (1.0, 200 / 255, 0.0)
-            red = (1.0, 0.0, 0.0)
-
             if not isinstance(max_r, (list, tuple)):
-                ax.axhline(y=max_r, xmin=0, xmax=1, color=red)
+                fig.add_hline(y=max_r, line_color="rgb(255,0,0)")
```

Comment:
Preferably use a configuration item or variable. We would like to be able to set colorblind-friendly traffic light colors.